hazelcast/src/test/java/com/hazelcast/map/MapListenerTest.java

```diff
@@ -16,82 +16,141 @@
 
 package com.hazelcast.map;
 
-import com.hazelcast.config.Config;
 import com.hazelcast.core.EntryEvent;
-import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.HazelcastInstanceAware;
 import com.hazelcast.core.IMap;
 import com.hazelcast.map.listener.EntryAddedListener;
 import com.hazelcast.map.listener.EntryRemovedListener;
 import com.hazelcast.map.listener.EntryUpdatedListener;
-import com.hazelcast.map.listener.MapListener;
 import com.hazelcast.query.SqlPredicate;
-import org.junit.Ignore;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.hazelcast.test.HazelcastTestSupport;
+import org.junit.Test;
 
 import java.io.Serializable;
-import java.util.Arrays;
 import java.util.Collection;
-import java.util.Map;
 import java.util.Random;
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicInteger;
 
-/**
- * A simple test that performs random puts, updates & removes on a map and counts the number of entries
- * and exits (with regards to the value space defined by the predicate) as performed by the map randomizer
- * and as observed on the listener side.
- */
-@Ignore("Not a JUnit test")
-public class MapListenerTest {
+import static org.junit.Assert.assertEquals;
+
+public class MapListenerTest extends HazelcastTestSupport {
```

Comment:
Missing `@RunWith` and `@Category`!

Reply:
Yeah, the missing annotations are really like a slap in the face! Fixed.